install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -451,8 +446,6 @@ spec:
 {{- if not (eq .Values.containerRuntime.integration "crio") }}
         - mountPath: /sys/fs/bpf
           name: bpf-maps
-{{- /* Required for wait-bpf-mount to work */}}
-          mountPropagation: HostToContainer
```

Comment:
Should this use `Bidirectional` also?

Reply:
Not really, the init container will not mount any bpf inside the container. It is only used to delete files, if they are available, from `/sys/fs/bpf`.